build.sbt

```diff
@@ -198,6 +197,16 @@ lazy val metacp = project
   .disablePlugins(BackgroundRunPlugin)
   .dependsOn(semanticdbJVM, cliJVM, ioJVM)
 
+lazy val symtab = project
+  .in(file("semanticdb/symtab"))
+  .settings(
+    publishableSettings,
+    description := "SemanticDB symbol table for Scala 2.x and Java classpaths"
+  )
+  // NOTE: workaround for https://github.com/sbt/sbt-core-next/issues/8
+  .disablePlugins(BackgroundRunPlugin)
```

Comment:
Why do wo we need this?

Reply:
Cargo culted from other project definitions. If we remove it we should do it for all projects in a separate PR.

Reply:
Let's open a ticket for that.